packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -100,7 +104,19 @@ export default function TemplatePanel() {
 					title={ decodeEntities( title ) }
 					icon={ CARD_ICONS[ record?.type ] ?? icon }
 					description={ decodeEntities( description ) }
-					actions={ <TemplateActions template={ record } /> }
+					actions={
+						<TemplateActions
+							postType={ postType }
+							postId={ postId }
+							className="edit-site-template-card__actions"
+							toggleProps={ { isSmall: true } }
```

Comment:
```suggestion
							toggleProps={ { size: 'small' } }
```

Although unrelated to this PR, we may want to avoid the prop that is internally deprecated. See #53560